docs/clustering/certificate-management.md

```diff
@@ -0,0 +1,51 @@
+# Certificate Management
+
+## Development
+
+Out of the box harperdb generates certificates that are used when HarperDB nodes are clustered together to securely share data between nodes. These certificates are meant for development/testing purposes. Because these certificates do not have CNs (Common Names) that will match the DNS/IP of the HarperDB node, the following settings (see the full [configuration file](../configuration.md) docs for more details) are defaulted & recommended for ease of development:
```

Comment:
```suggestion
Out of the box HarperDB generates certificates that are used when HarperDB nodes are clustered together to securely share data between nodes. These certificates are meant for development/testing purposes. Because these certificates do not have Common Names (CNs) that will match the Fully Qualified Domain Name (FQDN) of the HarperDB node, the following settings (see the full [configuration file](../configuration.md) docs for more details) are defaulted & recommended for ease of development:
```